include/cpptest-main.h

```diff
@@ -10,35 +10,11 @@
 
 #include "cpptest.h"
 
-#include <algorithm>
-#include <fstream>
-#include <functional>
```

Comment:
The header still requires the `<functional>` and `<string>` (which was missing before) includes

Reply:
Wups, you're right. Fixed.